qhub/stages/input_vars.py

```diff
@@ -95,7 +95,7 @@ def stage_02_infrastructure(stage_outputs, config):
                 {
                     "name": key,
                     "min_size": value["min_nodes"],
-                    "desired_size": value["min_nodes"],
+                    "desired_size": max(value["min_nodes"], 1),
```

Comment:
To follow the pattern I think we can update the min_node value directly from the init template [here](https://github.com/Quansight/qhub/blob/2687f13f54e3833234fc4282587be16521f0c84a/qhub/initialize.py#L162), and defined the values as we did for GCP in there..  What are your thoughts on this @tylerpotts ?